src/components/Api.js

```diff
@@ -0,0 +1,168 @@
+class Api {
+  constructor(options) {
+    this._authorization = "fd5203a5-1f57-4428-b210-42e565daa250";
```

Comment:
Needs correction: the token should be provided with the constructor arguments `options` (here you should save the whole headers object, not only the token,  and reuse them in all the api methods `this._headers = options.headers`)